src/Components/ErrorNotification.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  error: string | null;
+  isVisible: boolean;
+  onClose: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  error,
+  isVisible,
+  onClose,
+}) => (
+  <div
+    className={classNames(
+      'notification',
+      'is-danger',
+      'is-light',
+      'has-text-weight-normal',
+      { hidden: !isVisible },
```

Comment:
Here you can just check lenght of error string. You don't need to use additional props